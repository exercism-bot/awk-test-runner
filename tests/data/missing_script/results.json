{
  "version": 2,
  "status": "fail",
  "test-environment": {
    "gawk": "GNU Awk 5.1.0, API: 3.0 (GNU MPFR 4.1.0, GNU MP 6.2.1)",
    "bats": "Bats 1.7.0",
    "OS": "Ubuntu 22.04 LTS"
  },
  "tests": [
    {
      "name": "say hello",
      "status": "fail",
      "test_code": "run -127 bash one_passing.sh\n[ \"$status\" -eq 0 ]\n[ \"$output\" == \"Hello, World!\" ]",
      "message": "(in test file missing_script.bats, line 7)\n  `[ \"$status\" -eq 0 ]' failed\n"
    }
  ]
}
