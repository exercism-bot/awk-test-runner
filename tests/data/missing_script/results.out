1..1
not ok 1 say hello
# (in test file missing_script.bats, line 7)
#   `[ "$status" -eq 0 ]' failed
